examples/perl/ws_perl.pl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/perl
+
+#
+# /usr/local/websocketd/ws_perl.pl
+#
+# Once started up, this script keeps running and checking for the existance of files.  If a file exists, it 
+#   will send it to browser.
+#
+
+use strict;
+use Data::Dumper;
+use POSIX;
+
+# Autoflush output
+use IO::Handle;
+STDOUT->autoflush(1);
+
+my $cnt = 0;
+#
+# Stay open
+#
+while (1) {
+  $cnt++;
+
+  #
+  # Grab QUERY_STRING and parse for PHONE
+  #  Assumes a single parameter/value pair
+  #
+  my $QS     = $ENV{QUERY_STRING};
```

Comment:
needs some default .. e.g adding // "" would suffice, you probably also need use warnings; on top or just some sane "use vx.x" (I don't know what is common denominator for Perl would be now)